chia/wallet/cat_wallet/cat_wallet.py

```diff
@@ -59,6 +60,10 @@
 
 # This should probably not live in this file but it's for experimental right now
 
+CAT_MOD_HASH = CAT_MOD.get_tree_hash()
+CAT_MOD_HASH_HASH = Program.to(CAT_MOD_HASH).get_tree_hash()
```

Comment:
I don't understand this. `Program.to()` will create a single atom containing the hash. the tree hash of that atom will be different from `(q . <CAT_MOD>)`, right?

Reply:
Yes, the `q` addition is done in `generate_curry_and_treehash_f`.